pubsub/cloud-client/subscriber.py

```diff
@@ -207,6 +207,31 @@ def callback(message):
     # [END pubsub_subscriber_flow_settings]
 
 
+def receive_messages_synchronously(project, subscription_name):
+    """Pulling messages synchronously."""
+    # [START pubsub_subscriber_sync_pull]
+    subscriber = pubsub_v1.SubscriberClient()
+    subscription_path = subscriber.subscription_path(
+        project, subscription_name)
```

Comment:
Is there a public subscription name we can use instead of using this as a parameter? I recall I saw [one for taxis in the Dataflow templates quickstart](https://cloud-dot-devsite.googleplex.com/dataflow/docs/quickstarts/quickstart-templates).

If not, we should show a comment like `# subscription_name = '/does/a/sub/name/look/like/a/path?'`

Reply:
The `subscription_path` method will build `/projects/{project}/subscription/{subscription_name}`. 

Should we leave `subscription_name` as it is because it's just a string that's the subscription name?

Reply:
It's not clear that it's a string without some kind of comment or example. (Some APIs like Bigtable use bytes for names, so it's not obvious from context)